src/components/skeleton/Header/Header.js

```diff
@@ -57,4 +60,9 @@ export default class Header extends Component {
     // (solution: put some sort of icon at the top of the navigation so its non-clickabel?:)
     setTimeout(() => this.setState({isNavOpening: false}), 1000);
   }
+
+  _reverseName(name) {
+    var nameInParts = name.split(' ');
+    return nameInParts[1] + ' ' + nameInParts[0];
+  }
```

Comment:
Mja dat is het probleem niet echt, op mobile is dat nu het enige dat je nog ziet...

![2016-03-26 16_56_00-ttc erembodegem](https://cloud.githubusercontent.com/assets/2582910/14060941/ae856b26-f373-11e5-8265-62c5ef0cc997.png)

Zo ziet het er nu bij mij uit